libbeat/processors/add_kubernetes_metadata/config.go

```diff
@@ -50,15 +50,13 @@ type Enabled struct {
 
 type PluginConfig []map[string]config.C
 
-func defaultKubernetesAnnotatorConfig() kubeAnnotatorConfig {
-	return kubeAnnotatorConfig{
-		SyncPeriod:          10 * time.Minute,
-		CleanupTimeout:      60 * time.Second,
-		DefaultMatchers:     Enabled{true},
-		DefaultIndexers:     Enabled{true},
-		Scope:               "node",
-		AddResourceMetadata: metadata.GetDefaultResourceMetadataConfig(),
-	}
+func (c *kubeAnnotatorConfig) InitDefaults() {
```

Comment:
refers to the `Unpack` https://github.com/elastic/go-ucfg/blob/main/reify.go#L86-L87